week_10/진명재/week10_1043.java

```diff
@@ -0,0 +1,75 @@
+package week_10.진명재;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.*;
+import java.util.stream.IntStream;
+
+public class week10_1043 {
+    static ArrayList<Integer>[] arr;
+    public static void main(String[] args) throws Exception {
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+
+        StringTokenizer st = new StringTokenizer(br.readLine());
+
+        int n = Integer.parseInt(st.nextToken());
+
+        int m = Integer.parseInt(st.nextToken());
```

Comment:
```suggestion
        int N = Integer.parseInt(st.nextToken());

        int M = Integer.parseInt(st.nextToken());
```
상수는 대문자로 적는게 나중에 안헷갈리고 좋습니다.